xblock/test/django/test_request.py

```diff
@@ -80,12 +80,12 @@ def test_headers(self):
     def test_content_type(self):
         # Default charset
         self.assertEquals(
-            self._as_django(content_type='application/json')['Content-Type'],
-            'application/json; charset=UTF-8'
+            self._as_django(content_type='text/html')['Content-Type'],
+            'text/html; charset=UTF-8'
```

Comment:
@nedbat: All this test was intending to test was the behavior of the `charset` argument to `_as_django`. I think the main issue with the test as it was written as that it assumed it knew what charset webob was going to provide by default, and thus it broke when webob changed that default. I think the test should just be changed to assert that the content-type is set to the supplied content type, but ignore the charset, and then if the charset is set (as below), verify that `_as_django` sets the charset correctly.